packages/ui/src/Card.module.scss

```diff
@@ -0,0 +1,39 @@
+@use '../styles/constants' as c;
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  align-content: stretch;
```

Comment:
Did you mean `align-items`?

Reply:
Actually, that doesn't need to be there at all, thanks 🚀